openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-board.js

```diff
@@ -351,6 +351,7 @@ var Wb = function() {
 				}).parent().css('text-align', Settings.isRtl ? 'left' : 'right');
 				math.draggable({
 					scroll: false
+					, handle: '.ui-dialog-titlebar'
```

Comment:
Because of this the entire DIV was draggable, which meant on mobile/touch you can't click inside of the text. Nor the close icon. Nor the update button.